public/views/backup.html

```diff
@@ -30,11 +30,11 @@
     <div class="row" ng-show="(index.n>1 && index.m != index.n )">
       <div class="m10t columns size-14 text-gray">
         <span translate>
-          To restore this {{index.m}}-{{index.n}} shared wallet you will need
+          To restore this {{index.m}}-{{index.n}} <b>shared</b> wallet you will need
```

Comment:
perfect, thanks!

On Mon, Sep 28, 2015 at 4:46 PM, Gabriel Edgardo Bazán <
notifications@github.com> wrote:

> In public/views/backup.html
> <https://github.com/bitpay/copay/pull/3248#discussion_r40596215>:
>
> > @@ -30,11 +30,11 @@
> >      <div class="row" ng-show="(index.n>1 && index.m != index.n )">
> >        <div class="m10t columns size-14 text-gray">
> >          <span translate>
> > -          To restore this {{index.m}}-{{index.n}} shared wallet you will need
> > +          To restore this {{index.m}}-{{index.n}} <b>shared</b> wallet you will need
>
> In this line is the same phrase :
>
> gabrielbazan7@b128be5#diff-8d05805f74f9ce3f1a53ae0c3ffd4acbL46
> <https://github.com/gabrielbazan7/copay/commit/b128be58c17e1e4f64b75e1ec20891762a80f4a1#diff-8d05805f74f9ce3f1a53ae0c3ffd4acbL46>
>
> Also, I check that the translation exist in crowdin.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitpay/copay/pull/3248/files#r40596215>.
>



-- 
Matías Alejo Garcia
@ematiu
Roads? Where we're going, we don't need roads!